使用SSR获取动态失败

### 操作系统

Linux

### Python 版本

3.10.8

### NoneBot 版本

2.3.0

### Bilichat 版本

5.12.4

### 描述问题

 [ERROR] nonebot_plugin_bilichat | [Dynamic] 获取 UID RSS 动态失败: Request URL is missing an 'http://' or 'https://' protocol.
使用RSSHub自建源，源经RSS阅读器测试可以正常使用。

### 插件的配置项

DRIVER=~fastapi
ONEBOT_ACCESS_TOKEN=
SUPERUSERS=[""]
LOG_LEVEL=DEBUG

COMMAND_SEP=[""]
COMMAND_START=[""]
bilichat_cmd_start=""

bilichat_rss_base = "http://IP:PORT/"
bilichat_cd_time = 120
bilichat_neterror_retry = 3
bilichat_use_browser = True

### 截图或日志

```
09-02 17:15:43 [ERROR] nonebot_plugin_bilichat | [Dynamic] 获取 UID RSS 动态失败: Request URL is missing an 'http://' or 'https://' protocol.
Traceback (most recent call last):
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_transports/default.py", line 69, in map_httpcore_exceptions
    yield
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_transports/default.py", line 373, in handle_async_request
    resp = await self._pool.handle_async_request(req)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpcore/_async/connection_pool.py", line 167, in handle_async_request
    raise UnsupportedProtocol(
httpcore.UnsupportedProtocol: Request URL is missing an 'http://' or 'https://' protocol.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<string>", line 17, in <module>
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot/__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot/drivers/fastapi.py", line 186, in run
    uvicorn.run(
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/uvicorn/main.py", line 575, in run
    server.run()
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/uvicorn/server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/local/bin/python3/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot/message.py", line 476, in check_and_run_matcher
    await _run_matcher(
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot/message.py", line 428, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot/internal/matcher/matcher.py", line 850, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot/internal/matcher/matcher.py", line 825, in simple_run
    await handler(
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot/dependencies/__init__.py", line 94, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot_plugin_bilichat/commands/functions.py", line 30, in check_dynamic_v11
    if dyn := await fetch_last_dynamic(up):
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot_plugin_bilichat/lib/fetch_dynamic.py", line 32, in fetch_last_dynamic
    return await _fetchlast_rss(up.mid)
> File "/root/BiliBot/.venv/lib/python3.10/site-packages/nonebot_plugin_bilichat/lib/fetch_dynamic.py", line 98, in _fetchlast_rss
    response = await client.get(url)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_client.py", line 1801, in get
    return await self.request(
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_client.py", line 1574, in request
    return await self.send(request, auth=auth, follow_redirects=follow_redirects)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_client.py", line 1661, in send
    response = await self._send_handling_auth(
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_client.py", line 1689, in _send_handling_auth
    response = await self._send_handling_redirects(
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_client.py", line 1726, in _send_handling_redirects
    response = await self._send_single_request(request)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_client.py", line 1763, in _send_single_request
    response = await transport.handle_async_request(request)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_transports/default.py", line 372, in handle_async_request
    with map_httpcore_exceptions():
  File "/usr/local/bin/python3/lib/python3.10/contextlib.py", line 153, in __exit__
    self.gen.throw(typ, value, traceback)
  File "/root/BiliBot/.venv/lib/python3.10/site-packages/httpx/_transports/default.py", line 86, in map_httpcore_exceptions
    raise mapped_exc(message) from exc
httpx.UnsupportedProtocol: Request URL is missing an 'http://' or 'https://' protocol.
09-02 17:15:43 [INFO] nonebot | Matcher(type='message', module=nonebot_plugin_bilichat.commands.functions, lineno=19) running complete
09-02 17:15:43 [DEBUG] nonebot | Checking for matchers in priority 2...
09-02 17:15:43 [INFO] nonebot_plugin_bilichat | 查询 Text(text='checkdynamic UID', styles={}) 返回内容为空
```

--- a/nonebot_plugin_bilichat/lib/fetch_dynamic.py
+++ b/nonebot_plugin_bilichat/lib/fetch_dynamic.py
@@ -88,11 +88,8 @@ async def _fetchlast_grpc(up_mid: int, up_name: str) -> Dynamic | None:
 
 async def _fetchlast_rss(mid: int) -> Dynamic | None:
     try:
-        url = (
-            f"{plugin_config.bilichat_rss_base}bilibili/user/dynamic/{mid}"
-            + f"?key={plugin_config.bilichat_rss_key}"
-            if plugin_config.bilichat_rss_key
-            else ""
+        url = f"{plugin_config.bilichat_rss_base}bilibili/user/dynamic/{mid}" + (
+            f"?key={plugin_config.bilichat_rss_key}" if plugin_config.bilichat_rss_key else ""
         )
         async with httpx.AsyncClient() as client:
             response = await client.get(url)

--- a/nonebot_plugin_bilichat/subscribe/dynamic.py
+++ b/nonebot_plugin_bilichat/subscribe/dynamic.py
@@ -124,11 +124,8 @@ async def fetch_dynamics_grpc(up: Uploader):
 
 async def fetch_dynamics_rss(up: Uploader) -> Dynamic | None:
     try:
-        url = (
-            f"{plugin_config.bilichat_rss_base}bilibili/user/dynamic/{up.uid}"
-            + f"?key={plugin_config.bilichat_rss_key}"
-            if plugin_config.bilichat_rss_key
-            else ""
+        url = f"{plugin_config.bilichat_rss_base}bilibili/user/dynamic/{up.uid}" + (
+            f"?key={plugin_config.bilichat_rss_key}" if plugin_config.bilichat_rss_key else ""
         )
         async with httpx.AsyncClient() as client:
             response = await client.get(url)

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,7 @@
 [project]
 name = "nonebot-plugin-bilichat"
 
-version = "5.12.4"
+version = "5.12.5"
 
 description = "多种B站链接解析，视频词云，AI总结，你想要的都在 bilichat"
 authors = [

